CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
+# 0.2.0-dev
+
+- **BREAKING** `shortestPath` and `shortestPaths` now have one generic parameter
```

Comment:
We'd need to do this with the other methods as well - can you check on the feasibility?

Reply:
Just pushed an update. This works great for connected components. I don't think it applies to the async traverse logic, though. Thoughts?